src/components/tabs/tab-bar.tsx

```diff
@@ -1,48 +1,62 @@
 // tslint:disable-next-line:import-name
-import React, { Fragment, useState, useRef } from 'react';
+import React, { Fragment, useState, useRef, useEffect } from 'react';
 // tslint:disable-next-line:import-name
 import Tab from './tab';
 import uuid from 'uuid';
 
-export interface TabBarProps { addTab: void; children: any; }
+export interface TabBarProps {
+  reordable?: boolean; // booblean to active the reordable behavior of the tabs
```

Comment:
Fix spelling mistakes ('reorderable', 'boolean', 'closeable') and please make sure the prop names are appropriate for what they're used.

This will make it easier for people to contribute later on.

Reply:
the function of the props is on the comments, is there any of the props that you would think of a better name?

Reply:
They seem appropriate to me.